GRIB2_Template_4_126_ProductDefinitionTemplate_en.csv

```diff
@@ -0,0 +1,52 @@
+"Average, accumulation, or extreme values or other statistically processed values at a horizontal level or in a horizontal layer in a continuous or non-continuous time interval for radionuclides",40-41,Hours after reference time of data cut-off,(see Note 1),33,,,Operational
+"Average, accumulation, or extreme values or other statistically processed values at a horizontal level or in a horizontal layer in a continuous or non-continuous time interval for radionuclides",42,Minutes after reference time of data cut-off,,,,,Operational
+"Average, accumulation, or extreme values or other statistically processed values at a horizontal level or in a horizontal layer in a continuous or non-continuous time interval for radionuclides",43,Indicator of unit of time range (see Code table 4.4),,,4.4,,Operational
+"Average, accumulation, or extreme values or other statistically processed values at a horizontal level or in a horizontal layer in a continuous or non-continuous time interval for radionuclides",44-47,Forecast time in units defined by octet 43,(see Note 2),111,,,Operational
+"Average, accumulation, or extreme values or other statistically processed values at a horizontal level or in a horizontal layer in a continuous or non-continuous time interval for radionuclides",48,Type of first fixed surface (see Code table 4.5),,,4.5,,Operational
+"Average, accumulation, or extreme values or other statistically processed values at a horizontal level or in a horizontal layer in a continuous or non-continuous time interval for radionuclides",49,Scale factor of first fixed surface,,,,,Operational
+"Average, accumulation, or extreme values or other statistically processed values at a horizontal level or in a horizontal layer in a continuous or non-continuous time interval for radionuclides",50-53,Scaled value of first fixed surface,,,,,Operational
+"Average, accumulation, or extreme values or other statistically processed values at a horizontal level or in a horizontal layer in a continuous or non-continuous time interval for radionuclides",54,Type of second fixed surface (see Code table 4.5),,,4.5,,Operational
+"Average, accumulation, or extreme values or other statistically processed values at a horizontal level or in a horizontal layer in a continuous or non-continuous time interval for radionuclides",55,Scale factor of second fixed surface,,,,,Operational
+"Average, accumulation, or extreme values or other statistically processed values at a horizontal level or in a horizontal layer in a continuous or non-continuous time interval for radionuclides",56-59,Scaled value of second fixed surface,,,,,Operational
+"Average, accumulation, or extreme values or other statistically processed values at a horizontal level or in a horizontal layer in a continuous or non-continuous time interval for radionuclides",60-61,Year,,,,,Operational
```

Comment:
@sebvi should octets 60-67 include "of end of overall time interval" or did you remove it for a reason?